### PR TITLE
Warn logging if thread_ts is not a string

### PR DESCRIPTION
**Pull request by hi1280**
###  Summary

Warn logging if thread_ts argument of chat.postMessage, chat.postEphemeral, chat.scheduleMessage and files.upload is not a string.

fix: #1196 

### Requirements (place an `x` in each `[ ]`)

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/node-slack-sdk/blob/master/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

### CODE REVIEW

File: packages/web-api/src/WebClient.ts [hunk 1]
```diff
@@ -645,3 +647,18 @@ function warnIfFallbackIsMissing(method: string, logger: Logger, options?: WebAP
...
+ * @param method api method being called
+ * @param logger instance of web clients logger
+ * @param options arguments for the Web API method
+ */
+function warnIfThreadTsIsNotString(method: string, logger: Logger, options?: WebAPICallOptions): void {
+  const targetMethods = ['chat.postEphemeral', 'chat.postMessage', 'chat.scheduleMessage', 'files.upload'];
+  const isTargetMethod = targetMethods.includes(method);
+
+  if (isTargetMethod && options?.thread_ts !== undefined && typeof options?.thread_ts !== 'string') {
+    logger.warn('The thread_ts argument need to be a string.');
```
Comment (seratch) [inline packages/web-api/src/WebClient.ts:662]: Can you include the method name in the log message like we do in `warnIfFallbackIsMissing` method?

```suggestion
    logger.warn(`The given \`thread_ts\` value in the request payload for a ${method} call is a float value. We highly recommend using a string value instead.');
```

**Review by seratch — CHANGES_REQUESTED** [review]
Thanks for taking the time to make this contribution! It's pretty nice to have the tests covering all the patterns 👍  One thing, I have a suggestion about the log message to make it more informative and consistent with the similar `warnIfFallbackIsMissing` method.

**Comment by stevengill** [comment]
Thanks for sending this in @hi1280! I'd say just make the suggestion by @seratch and this looks good to merge!

File: packages/web-api/src/WebClient.ts (hunk 1 above)
Comment (hi1280) [reply packages/web-api/src/WebClient.ts]: @seratch Thank you for your review. I fixed the thing you suggested.

**Review by seratch — APPROVED** [review]
Thanks!

**Pull request merged**